src/lib.rs

```diff
@@ -405,7 +406,8 @@ struct NodeData<'input> {
 
 #[derive(Clone, Debug)]
 struct AttributeData<'input> {
-    name: ExpandedNameIndexed<'input>,
+    /// Indexes into `ExpandedNames::values`
+    name_idx: u32,
```

Comment:
It is never exposed in the crate's API so it would only be us who have to keep this straight, but yes a newtype replace the comment and prevent accidentally indexing into the wrong collection. We should probably start with `namespace_idx` in #80 though.

@RazrFalcon Since it's your project: What are you preferences w.r.t using newtypes to mark the target of the indexes?

Reply:
I think it would be a good idea. A bit more verbose, but potentially safer.

Reply:
This is part of #84